k8s/web/deployment.yml

```
@@ -0,0 +1,109 @@
+        - name: static-volume
+          persistentVolumeClaim:
+            claimName: static-pvc
+        - name: shared-data
+          persistentVolumeClaim:
+            claimName: shared-data-pvc
+      initContainers:
+      - name: init-migrate
+        image: ghcr.io/khulnasoft/reconpoint/web:latest
+        command: ["sh", "-c", "mkdir -p /usr/src/shared-app && cp -r /usr/src/app/* /usr/src/shared-app && python3 manage.py migrate && python3 manage.py collectstatic --noinput"]
+        env:
+          - name: POSTGRES_HOST
```

yamllint (1.35.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)